keepercommander/commands/enterprise.py

```diff
@@ -3027,7 +3027,8 @@ def convert_value(self, field, value, **kwargs):
             return ''
 
         if field == "created":
-            dt = datetime.datetime.utcfromtimestamp(int(value)).replace(tzinfo=datetime.timezone.utc).astimezone(tz=None)
+            parsed_time = datetime.datetime.strptime(value, '%Y-%m-%d %H:%M:%S.%f')
```

Comment:
this change breaks
audit-report --report-type=raw 
command

Reply:
Thanks Sergey.  Are there any other tests I should be running?

Reply:
I think that is it.
If we check the type and do not change `int` branch that would be enough.
At least we do not break the existing functionality